buf.gen.yaml

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env -S buf generate buf.build/authzed/api:3444bd6f843d49928d6a1226965d88b7 --template
+version: "v1beta1"
```

Comment:
Is this supposed to be `v1alpha1`?

Reply:
Nope -- this is the version of the schema for this file. Joey actually thought the same thing earlier today.